packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -91,6 +97,10 @@ class GoogleSignInAccount implements GoogleIdentity {
     if (response['idToken'] == null) {
       response['idToken'] = _idToken;
     }
+
+    if (response['serverAuthCode'] == null) {
```

Comment:
nits: `response['serverAuthCode'] ??=_serverAuthCode;`

Reply:
It's cool